tests/unit/test_masks.py

```diff
@@ -177,22 +180,30 @@ def it_knows_its_mask_tile(self, request):
     def it_knows_its_mask_tile_and_supports_custom_filters(self, request):
         tile = Tile(PILIMG.RGBA_COLOR_500X500_155_249_240, None, None)
         custom_filters = [CustomFilterForTest()]
-        custom_filter_call_ = method_mock(request, CustomFilterForTest, "__call__")
-        # We need to account for the fact padding is added before filtering
-        expected_mask = np.zeros((510, 510), dtype=np.bool)
-        custom_filter_call_.return_value = expected_mask
+        expected_mask = [
+            [True, True],
+            [False, True],
+        ]
```

Comment:
please use a np array here to be consistent with the real `calculate_tissue_mask` return type